topics/transcriptomics/tutorials/limma-voom_gene_set_testing/tutorial.md

```diff
@@ -0,0 +1,283 @@
+---
+layout: tutorial_hands_on
+title: RNA-seq genes to pathways
+zenodo_link: "https://zenodo.org/record/2491305"
+tags:
+  - mouse
+questions:
+  - "What are the differentially expressed pathways in the mammary gland of pregnant versus lactating mice?"
+objectives:
+  - "Identification of differentially expressed pathways"
+time_estimation: "2h"
+key_points:
+  - "Multiple methods can be used to help identify differentially expressed pathways"
+contributors:
+  - mblue9
+  - bphipson
```

Comment:
Do they all agree? If so you need to add them to the https://github.com/galaxyproject/training-material/blob/master/CONTRIBUTORS.yaml as well.

Reply:
I already added them in some weeks ago.